### ISSUE TITLE
Tags should have border options to improve contrast ratio

### ISSUE THREAD

**Issue by rsimoens** [vtex/styleguide#522]
**Describe the bug**
Our low prominence tags examples on styleguide are not even close to accessibility good practices. 

**To Reproduce**
Color contrast can be tested on sites like https://webaim.org/resources/contrastchecker/

**Expected behavior**
We must guarantee or at least try to top the AA level of accessibility guidelines (3:1 contrast ratio in this case).
A version with borders and white background would have better results.

**Screenshots**
Now this is our examples contrast ratios on styleguide:
![image](https://user-images.githubusercontent.com/10401309/52137631-2acfea00-2632-11e9-8ab0-ff93a4424800.png)

**Additional context**
With borders, we could achieve this style and contrast ratios:
![image](https://user-images.githubusercontent.com/10401309/52137675-463af500-2632-11e9-984f-0cdc239a8ee2.png)

**Comment by cmdalbem** [comment]
Closed by https://github.com/vtex/styleguide/pull/542
